src/app/shared/components/route-selector/route-selector.component.ts

```
@@ -194,59 +201,68 @@ export class RouteSelectorComponent implements OnInit, ControlValueAccessor, OnD
       });
 
-      data.mileageLocations?.forEach((mileageLocation) => {
+      data.mileageLocations?.forEach((mileageLocation: MileageLocation) => {
         this.mileageLocations.push(
           new UntypedFormControl(mileageLocation || {}, this.mileageConfig.location_mandatory && Validators.required)
         );
       });
 
       this.form.patchValue({
-        distance: parseFloat(data.distance),
+        distance: parseFloat(String(data.distance)),
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)